swagger-spec/search/projects.yaml

```diff
@@ -1 +1,159 @@
 # /search/projects/
+get:
+  summary: Search the OSF for projects
+  description: >-
+    This endpoint returns projects that have been found by the given Elasticsearch query. All projects returned
+    are serialized a projects.
```

Comment:
Same typo - Either 'are serialized as projects' or 'are serialized projects' or 'are serialized' should be fine.